### PR TITLE
Fix exporting-server sample for java server and html client

### PR DESCRIPTION
**Pull request by luispandolfi**

### CODE REVIEW

File: Exporting-Server/server/spring-boot-jersey/src/main/java/com/server/reveal/RevealDashboardProvider.java [hunk 1]
```diff
@@ -12,8 +12,7 @@ public class RevealDashboardProvider implements IRVDashboardProvider {
 
     @Override
     public InputStream getDashboard(IRVUserContext userContext, String dashboardId) throws IOException {
-        InputStream dashboardStream = new FileInputStream("dashboards/" + dashboardId + ".rdash");
```
Comment (brianlagunas) [inline Exporting-Server/server/spring-boot-jersey/src/main/java/com/server/reveal/RevealDashboardProvider.java:15]: Why did you move the dashboard and change the loading strategy?

Comment (luispandolfi) [reply Exporting-Server/server/spring-boot-jersey/src/main/java/com/server/reveal/RevealDashboardProvider.java:15]: I used Eclipse to run the server. Was not loading the dashboard. Is there any setup I have to do to load it?

Comment (brianlagunas) [reply Exporting-Server/server/spring-boot-jersey/src/main/java/com/server/reveal/RevealDashboardProvider.java:15]: I don't use Eclipse. I use VS Code and it works there. Loading dashboards from a resource is not the same as loading from a folder on the server. I'm closing this since it doesn't really fix anything.
